app/assets/stylesheets/form.scss

```diff
@@ -0,0 +1,53 @@
+.form-link {
+  padding: 4px;
+  float: right;
+  a {
+    font-size: 14px;
+  }
+}
+
+.form-container {
```

Comment:
Let's keep this class name in mind in case we end up making any other forms on future iterations. We may want different styling, in which case maybe name this new-session-user-form or something.